app/controllers/editorial/news_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Editorial
```

Comment:
OK. So this is not possible to achieve with type-based form objects & form views? Or just not worth the time investment currently?